loki/batch/sgraph.py

```diff
@@ -333,6 +375,33 @@ def successors(self, item, item_filter=None):
                     successors += (child,)
         return successors
 
+    def subsgraph(self, item, item_filter=None):
```

Comment:
Im a little undecided about the naming, because the word looks a little weird and I have more than once read "_subs-graph_". Python `snake_case` would suggest to use `sub_sgraph` - what's your thoughts on this @MichaelSt98 @mlange05 ? Open to other suggestions or being overruled here.

Reply:
Hmm, naming is hard. Mh intuition would be to stick with `subgraph` to avoid weirdness, but snake case would be fine too for me.